From c0584ec188166c2ad5b4e9a733977241319cfccb Mon Sep 17 00:00:00 2001
From: Denys Dmytriyenko <denys@konsulko.com>
Date: Wed, 26 May 2021 10:19:05 -0500
Subject: linuxptp: update bbappend against latest upstream

Back-porting upstream patch dc1101ed36094749219285378a73f660255e3cbd
for dunfell. The dunfell linux-ptp is still at 2.0. This forks linuxptp
to 3.0 to update it for the linux ptp fixes done on meta-openembedded

Signed-off-by: Denys Dmytriyenko <denys@konsulko.com>
Signed-off-by: Yogesh Siraswar <yogeshs@ti.com>
Cc: Lokesh Vutla <lokeshvutla@ti.com>
---
 meta-arago-distro/recipes-connectivity/linuxptp/linuxptp_%.bbappend | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/meta-arago-distro/recipes-connectivity/linuxptp/linuxptp_%.bbappend b/meta-arago-distro/recipes-connectivity/linuxptp/linuxptp_%.bbappend
index b0552aaf..e4fb109a 100644
--- a/meta-arago-distro/recipes-connectivity/linuxptp/linuxptp_%.bbappend
+++ b/meta-arago-distro/recipes-connectivity/linuxptp/linuxptp_%.bbappend
@@ -1,13 +1,13 @@
 PR_append = ".arago0"
-PV_append = "+git${SRCPV}"
+PV = "3.0+git${SRCPV}"
 
 BRANCH = "ti-linuxptp"
-SRCREV = "a06698a12e828a0372955c2d9c8d6604d8326811"
+SRCREV = "f155c21283a0421f5e1e0b5e850c8a9a71d72453"
 
 SRC_URI = " \
     git://git.ti.com/processor-sdk/linuxptp.git;branch=${BRANCH} \
     file://build-Allow-CC-and-prefix-to-be-overriden.patch \
-    file://no-incdefs-using-host-headers.patch \
+    file://Use-cross-cpp-in-incdefs.patch \
 "
 
 S = "${WORKDIR}/git"
-- 
cgit v1.2.3-13-gbd6f

